pkgs/tools/virtualization/ovftool/default.nix

```diff
@@ -55,9 +54,6 @@ stdenv.mkDerivation rec {
 
   src = ovftoolSource;
 
-  # Maintainers: try downloading a NixOS OVA and run the following to test:
-  # `./result/bin/ovftool https://channels.nixos.org/nixos-unstable/latest-nixos-x86_64-linux.ova nixos.ovf`
-  # Some dependencies are not loaded until operations actually occur!
```

Comment:
Is this advice no longer valid?

Reply:
Yeah, this was prior to us having an installCheckPhase.